subgraphs/aura-finance/src/prices/index.ts

```diff
@@ -0,0 +1,149 @@
+import * as constants from "./common/constants";
+import { CustomPriceType } from "./common/types";
+import { getCurvePriceUsdc } from "./routers/CurveRouter";
+import { getTokenPriceFromOneInch } from "./oracles/1InchOracle";
+import { getTokenPriceFromAaveOracle } from "./oracles/AaveOracle";
+import { getTokenPriceFromChainLink } from "./oracles/ChainLinkFeed";
+import { getTokenPriceFromYearnLens } from "./oracles/YearnLensOracle";
+import { getPriceUsdc as getPriceUsdcUniswap } from "./routers/UniswapRouter";
+import { log, Address, BigDecimal, dataSource } from "@graphprotocol/graph-ts";
+import { getPriceUsdc as getPriceUsdcSushiswap } from "./routers/SushiSwapRouter";
+import { getTokenPriceFromSushiSwap } from "./calculations/CalculationsSushiswap";
+import { getTokenPriceFromCalculationCurve } from "./calculations/CalculationsCurve";
+
+export function getUsdPricePerToken(tokenAddr: Address): CustomPriceType {
+  // Check if tokenAddr is a NULL Address
+  if (tokenAddr.toHex() == constants.ZERO_ADDRESS_STRING) {
+    return new CustomPriceType();
+  }
+
+  let network = dataSource.network();
+
+  if (constants.MAI_STABLECOIN_MAP.get(network)! == tokenAddr) {
```

Comment:
The [contract for MAI-USDC_SP pool's lp-token](https://etherscan.io/address/0xf3aeb3abba741f0eece8a1b1d2f11b85899951cb) is not public. The price calculated for this lp token, using logic used for other weighted pools or stable pools, does not match the offiicial website number. 
Since it's a stablecoin, we'll assume it to be dollar price.